mirage/factories/project.ts

```diff
@@ -46,7 +46,9 @@ export default Base.extend({
   },
 
   platformIconClass() {
-    switch (this.platform) {
+    const platform = this.platform as unknown as number;
```

Comment:
is `unknown` required here ?

Reply:
Yes. You can't cast a function directly to a number.